src/common/components/nav/desktopNav.tpl.html

```diff
@@ -109,8 +109,7 @@ <h5>{{country.title}}</h5>
             <a uib-dropdown-toggle>
               <i class="fal fa-shopping-cart"></i>
             </a>
-            <div class="pull-right" uib-dropdown-menu>
-              <span class="triangle"></span>
+            <div id="nav-cart-dropdown" uib-dropdown-menu>
```

Comment:
Make this change to this file too: src/common/components/nav/navCartIcon.tpl.html

desktopNav.tpl.html will go away eventually.